providers/implementations/signature/eddsa_sig.c

```diff
@@ -574,12 +597,17 @@ static int eddsa_set_ctx_params(void *vpeddsactx, const OSSL_PARAM params[])
         }
     }
 
+    if (!OSSL_FIPS_IND_SET_CTX_PARAM(peddsactx, OSSL_FIPS_IND_SETTABLE0, params,
```

Comment:
move this check up.. we should always do this first.

Reply:
Why?

Reply:
because if the params are all set in the same call it should be done first -before any other sets that use that value.

Reply:
Fair point but the set_ctx_params isn't doing any checking (that happens in the verify calls).
Anyway, I moved it already.